dist/api_data.py

```diff
@@ -910,6 +932,41 @@ def __ge__(self, other):
         type='category', subconfig=
         statistics_log_configuration_common)
 ]
+
+tiered_storage_configuration_common = [
+    Config('auth_timeout', 0, r'''
+        time in seconds that the authentication token is valid''',
+        min='0', max='100000'),
+    Config('auth_token', '', r'''
+        authentication token string'''),
+    Config('local_retention', '5', r'''
+        minutes to retain data on tiered storage on the local tier for
+        faster read access''',
+        min='0', max='120'),
```

Comment:
I think I'd rather see seconds than minutes.  I don't think minutes is used elsewhere in the API.  If we want a convenient way to mention minutes, let's build it into the config parser (like it understands the 1k means 1*2^10, we could have 10min or 10hr).

Reply:
Good point. Even long operations like checkpoint and sweep use seconds. I'll change that. I don't think we need to build minutes/hours into the config parser right now.